### DISCUSSION TITLE
The "Used by" of template repository

### DISCUSSION THREAD

**Discussion by skyleaworlder** [community/community#55499] [Repositories]
### Select Topic Area

Product Feedback

### Body

I think that the current template repositories lack a way for the maintainers to actively know the usage of generated repositories through the original repository. This makes it difficult for the template repository maintainers to proactively understand the situation of the users, and I believe that the number of generated repositories is also an important indicator for users to choose a template repository.

BTW, I think the "Used by" is very suitable for this responsibility.

**Comment by critesjosh** [comment]
I agree. I would love to see something like this.

**Comment by tmsdkeys** [comment]
+1. I'm creating a template repo as it makes the most sense for my project but traditional KPIs like number of clones/forks would fall away afaik

would be good to have an alternative way for this and more general management options as OP suggested
